packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1233,11 +1221,20 @@
 		<path d="m5 20c-0.55 0-1.0208-0.1958-1.4125-0.5875s-0.5875-0.8625-0.5875-1.4125v-12c0-0.55 0.19583-1.0208 0.5875-1.4125s0.8625-0.5875 1.4125-0.5875h14c0.55 0 1.0208 0.19583 1.4125 0.5875s0.5875 0.8625 0.5875 1.4125v12c0 0.55-0.1958 1.0208-0.5875 1.4125s-0.8625 0.5875-1.4125 0.5875h-14zm2-5h3c0.2833 0 0.5208-0.0958 0.7125-0.2875s0.2875-0.4292 0.2875-0.7125v-1h-1.5v0.5h-2v-3h2v0.5h1.5v-1c0-0.28333-0.0958-0.52083-0.2875-0.7125s-0.4292-0.2875-0.7125-0.2875h-3c-0.28333 0-0.52083 0.09583-0.7125 0.2875s-0.2875 0.42917-0.2875 0.7125v4c0 0.2833 0.09583 0.5208 0.2875 0.7125s0.42917 0.2875 0.7125 0.2875zm7 0h3c0.2833 0 0.5208-0.0958 0.7125-0.2875s0.2875-0.4292 0.2875-0.7125v-1h-1.5v0.5h-2v-3h2v0.5h1.5v-1c0-0.28333-0.0958-0.52083-0.2875-0.7125s-0.4292-0.2875-0.7125-0.2875h-3c-0.2833 0-0.5208 0.09583-0.7125 0.2875s-0.2875 0.42917-0.2875 0.7125v4c0 0.2833 0.0958 0.5208 0.2875 0.7125s0.4292 0.2875 0.7125 0.2875z"/>
 	</symbol>
 
-
 	<symbol id="kv-tag-alarm"
 		xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
 		<path d="M13.04 11.9V10.45H10.96V11.9H13.04Z"/>
 		<path fill-rule="evenodd" clip-rule="evenodd" d="M5 4C3.89543 4 3 4.89543 3 6V18C3 19.1046 3.89543 20 5 20H19C20.1046 20 21 19.1046 21 18V6C21 4.89543 20.1046 4 19 4H5ZM9.40002 15.1H10.96V13.15H13.04V15.1H14.6V9.93336C14.6 9.64058 14.5004 9.39516 14.301 9.19711C14.1017 8.99905 13.8547 8.90002 13.56 8.90002H10.44C10.1454 8.90002 9.89836 8.99905 9.69902 9.19711C9.49969 9.39516 9.40002 9.64058 9.40002 9.93336V15.1Z"/>
 	</symbol>
 
+	<symbol id="kv-radio-btn" viewBox="0 0 24 24"
+		xmlns="http://www.w3.org/2000/svg">
+		<path d="m12 22c-1.3833 0-2.6833-0.2627-3.9-0.788-1.2167-0.5247-2.275-1.237-3.175-2.137s-1.6123-1.9583-2.137-3.175c-0.52533-1.2167-0.788-2.5167-0.788-3.9s0.26267-2.6833 0.788-3.9c0.52467-1.2167 1.237-2.275 2.137-3.175s1.9583-1.6127 3.175-2.138c1.2167-0.52467 2.5167-0.787 3.9-0.787s2.6833 0.26233 3.9 0.787c1.2167 0.52533 2.275 1.238 3.175 2.138s1.6123 1.9583 2.137 3.175c0.5253 1.2167 0.788 2.5167 0.788 3.9s-0.2627 2.6833-0.788 3.9c-0.5247 1.2167-1.237 2.275-2.137 3.175s-1.9583 1.6123-3.175 2.137c-1.2167 0.5253-2.5167 0.788-3.9 0.788zm0-2c2.2333 0 4.125-0.775 5.675-2.325s2.325-3.4417 2.325-5.675c0-2.2333-0.775-4.125-2.325-5.675s-3.4417-2.325-5.675-2.325c-2.2333 0-4.125 0.775-5.675 2.325s-2.325 3.4417-2.325 5.675c0 2.2333 0.775 4.125 2.325 5.675s3.4417 2.325 5.675 2.325z" />
```

Comment:
This was parsed by svgo?

Reply:
I used a svg minifier, still came out big like that tho :/

Reply:
use [svgo](https://github.com/svg/svgo)

Reply:
Now it should be :ok_hand: